system-test/bigquery.ts

```diff
@@ -1035,6 +1035,21 @@ describe('BigQuery', () => {
             );
           });
 
+          it('should work with empty arrays', done => {
```

Comment:
This could be far shorter if done with async style:

```js
it('should work with empty arrays', async () => {
  const [rows] = await bigquery.query({
    query: 'SELECT * FROM UNNEST (?)',
    params: [[]],
    types: [['INT64']],
  });
  assert.strictEqual(rows.length, 0);
});
```

For new code, I would prefer we avoid callbacks, and lean on async/await style code

Reply:
Okay, I'll start using async/await instead.